### DISCUSSION TITLE
Signin page not working

### DISCUSSION THREAD

**Discussion by chillyjim** [jbergler/hass-ttlock#145] [Q&A] [answered]
## Describe the bug

I'm probably just doing something wrong, but I can't figure it out.

On the second setup screen, when I enter my login information for the ttlock application, I just get "Invalid account or Invalid password."

### To Reproduce

1. Add the integration
2. enter the API ID/Key
3. Next page

<img width="593" alt="image" src="https://github.com/user-attachments/assets/a49097b8-5207-46d9-bbee-405370e981ef">

### Expected behavior

Integration should be added.

**Comment by jbergler** [comment]
Hey @chillyjim - did you follow the instructions in the readme?

Please refer to these two pages and close this issue if you were able to fix the problem.
https://github.com/jbergler/hass-ttlock/blob/develop/README.md
https://github.com/jbergler/hass-ttlock/issues/55#issuecomment-1610036292

**Comment by chillyjim** [comment]
I did, and I have tried my app (iPhone) & my euopen.ttlock.com logins.
There isn't much in the readme except get your API key and follow the bouncing ball.

**Comment by taimaiduc** [comment]
The same thing for me. Can not login

**Comment by johnporcaro** [comment]
I have the same issue. Verified my app info on the website, but using HA to log in throws Invalid account or Invalid password error.

**Comment by SonicBee** [comment]
Same issue, I cant get the Username/Password to be accepted even after verifying that it is correct.

**Comment by jbergler** [comment, marked as answer]
What username & password are you using?

It needs to be the username/password you use to log into the ttlock app. Not the login for euopen.ttlock.com

**Comment by chillyjim** [comment]
It took a while but I was finally able to get in with my phone number not my email which is what the app listed as my username. I did change my password I don’t know if that made any difference.

**Comment by taimaiduc** [comment]
invalid client_id
This is the message reponnse

**Comment by johnporcaro** [comment]
Thanks. Tried again today and it seems to be working!


Sent from my mobile phone

On Oct 10, 2024, at 1:08 AM, Jonas Bergler ***@***.***> wrote:

﻿

What username & password are you using?

It needs to be the username/password you use to log into the ttlock app. Not the login for euopen.ttlock.com

**Comment by quynv123** [comment]
Dear All, 
I have the same issue like this. I use the same email address and password for the TTLock on my phone and the Open Platform account, but I still can not log into this interface to go through step 4 in this instruction: https://github.com/jbergler/hass-ttlock/blob/develop/README.md
I have tried with my phone number, I have tried to remove TTlock from HACS and re-download it, but it still not works. 
Please instruct me how to figure out the problem. 
Thank you

**Reply by N3rd7** [reply]
Your issue could be the same as mine: if for any reason you did not enter the OAuth correctly, uninstalling from HACS will not clear this information. You need to go to Settings/Devices then click on the top right 3 dots, open application credentials, then remove the credentials for TTLock. Then go back to add TTLock, and make sure that in the OAuth window, you enter the credentials from the app you created on https://open.ttlock.com/manager, and not your usual application credentials. Hope this works for you!

**Reply by quynv123** [reply]
Hi N3rd7, 
Thank you for the instruction. It works for me. The key point is 3 dots on the top right.
Quy

**Comment by taimaiduc** [comment]
You need to remove old config

![image](https://github.com/user-attachments/assets/6df55c80-18d0-4797-b2f7-209cc6c00af1)
